vsintegration/Vsix/VisualFSharpOpenSource/VisualFSharpOpenSource.csproj

```diff
@@ -56,13 +56,17 @@
     <MicroBuildAssemblyVersion>15.4.1.0</MicroBuildAssemblyVersion>
     <MicroBuildAssemblyFileLanguage>cs</MicroBuildAssemblyFileLanguage>
   </PropertyGroup>
+  
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <IncludeDebugSymbolsInVSIXContainer>true</IncludeDebugSymbolsInVSIXContainer>
```

Comment:
This looks strange to me. Can someone tell me why this is an _addition_? Shouldn't _Debug_ symbols on for the Debug build?